packages/main/src/Menu.ts

```diff
@@ -535,6 +539,40 @@ class Menu extends UI5Element {
 		this._parentItemsStack.push(item);
 	}
 
+	startOpenTimeout(item: MenuItem, opener: OpenerStandardListItem, hoverId: string) {
+		// If theres already a timeout, clears it
+		this.clearOpenTimeout(item);
+
+		// Sets the new timeout
+		this.hoverTimeouts[item.id] = window.setTimeout(() => {
+			this._prepareSubMenuDesktopTablet(item, opener, hoverId);
+		}, 300);
+	}
+
+	startCloseTimeout(item: MenuItem) {
+		// If theres already a timeout, clears it
+		this.clearCloseTimeout(item);
+
+		// Sets the new timeout
+		this.unhoverTimeouts[item.id] = window.setTimeout(() => {
+			this._closeItemSubMenu(item);
+		}, 400);
```

Comment:
Is there a reason to have different timeouts for opening and closing?
If yes, this also should be a constant.